packages/stream-model-handler/src/model-handler.ts

```diff
@@ -106,7 +106,8 @@ export class ModelHandler implements StreamHandler<Model> {
     const streamId = await StreamID.fromGenesis('model', commitData.commit)
     const { controllers, model } = payload.header
     const controller = controllers[0]
-    const modelStreamID = StreamID.fromBytes(model)
+    const modelStreamID =
```

Comment:
the commit should never have the streamid encoded as a string, they should always be bytes.  We can be permissive with types for higher-level APIs that are exposed to end users, but for the IPLD data structures that are actually stored in the network we want them to be very strictly typed and encoded in the most efficient way possible

Reply:
Nice catch!

Reply:
Suggestion is to move this logic into https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/common/src/utils/stream-utils.ts#L78? Was hesitant to move this into that location, since it seems model agnostic. We could also add functionality to the model handlers to `sanitize` a message, so it exists outside of this method.